notebooks/data-sources/Telemetry/telemetry_EDA.ipynb

```diff
@@ -15,11 +15,11 @@
   },
```

Comment:
<p>We are able to get the status of past builds from the historical testgrid data that we're collecting. That is how we are calculating build pass/fail metrics over time. </p><p>However, I am not sure if we are able to get the build ID from the Testgrid data. So getting it from the prow logs makes sense. Maybe we should modify the markdown accordingly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/470/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Thanks for the feedback @oindrillac :) . I have made changes as per your comment.